scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -315,7 +316,7 @@ task GetNumSamplesLoaded {
 
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
         bq query --location=US --project_id=~{project_id} --format=csv --use_legacy_sql=false \
-        "SELECT COUNT(*) as num_rows FROM ~{fq_sample_table} WHERE is_loaded = true" > num_rows.csv
+        "SELECT COUNT(*) as num_rows FROM ~{fq_sample_table}" > num_rows.csv
```

Comment:
I'm confused about why we don't need to check this anymore in the (default) case where the table is `sample_info`

Reply:
this must have been a hold over from when I thought I was going to remove this field and forgot to put this back in. adding back.